client/handle_http.go

```diff
@@ -2153,6 +2153,14 @@ func uploadObject(transfer *transferFile) (transferResult TransferResults, err e
 		ioreader = ap
 		sizer = ap
 	} else {
+
+		if fileInfo.IsDir() {
+			log.Errorln("The given path", transfer.localPath, "is a directory. Use --recursive or -r flag if using the pelican object command for directories")
```

Comment:
```suggestion
			log.Errorln("The given path", transfer.localPath, "is a directory. Use the sync command or the --recursive or -r flag if using the pelican object command for directories")
```

Reply:
I simplified the message to: `the provided path <given-path> is a directory, but a file is expected`. I felt it was unnecessary to list all possible ways to handle directories, as new methods might be introduced in the future. Let me know if you disagree, and I can revise it.

https://github.com/PelicanPlatform/pelican/pull/1707/files#diff-4b3ff5f5414bcb82345ca02dead65d984f23d0737392da402d2517b8f192ce61R2158